app/utils/analytics.js

```diff
@@ -0,0 +1,17 @@
+
+export function analytics(eventData) {
+  if(eventData.value) {
+    _paq.push(['trackEvent', eventData.category, eventData.action, eventData.name, eventData.value])
+  } else {
+    _paq.push(['trackEvent', eventData.category, eventData.action, eventData.name])
+  }
+}
+
+export function initializeMatomoTagManager(containerID) {
+  var _mtm = window._mtm = window._mtm || [];
+  _mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+  var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+  g.type='text/javascript';
+  g.id='matomo-tag-manager';
+  g.async=true; g.src=`https://cdn.matomo.cloud/nycplanning.matomo.cloud/container_${containerID}.js`; s.parentNode.insertBefore(g,s);
```

Comment:
It looks like this section is creating a script tag:

```javascript
<script type='text/javascript' id='matomo-tag-manager' async=true src=`https://cdn.matomo.cloud/nycplanning.matomo.cloud/container_${containerID}.js`/>
```

Would it be simpler to move the fully constructed tag to the end of the [document body tags](https://github.com/NYCPlanning/ae-cp-map/blob/6a436307839908147d88cef01674ec59fdf3b086/app/root.tsx#L137)?

Reply:
I don't know if there is functionally any difference between the two, but I would prefer to leave the code as-is to maintain consistency with the code snippet they provide, and across our different repos.